include/openssl/core_names.h

```diff
@@ -73,6 +73,13 @@ extern "C" {
 /* For passing the AlgorithmIdentifier parameter in DER form */
 #define OSSL_CIPHER_PARAM_ALG_ID             "alg_id_param" /* octet_string */
 
+#define OSSL_CIPHER_PARAM_TLS1_MULTIBLOCK_AAD_IN "tlsmultiaad" /* octet_string */
+#define OSSL_CIPHER_PARAM_TLS1_MULTIBLOCK_ENC_IN "tlsmultienc" /* octet_string */
+#define OSSL_CIPHER_PARAM_TLS1_MULTIBLOCK_ENC    "tlsmultienc" /* octet_string */
```

Comment:
These two seem to have the same string value

Reply:
updated and added docs